plugin/tiff/pom.xml

```diff
@@ -45,5 +45,11 @@
           <version>${project.version}</version>
           <scope>test</scope>
       </dependency>
+    <dependency>
+      <groupId>it.geosolutions.jaiext.utilities</groupId>
+      <artifactId>jt-utilities</artifactId>
+      <version>1.1.23</version>
+      <scope>test</scope>
```

Comment:
Not sure to remember. where it is needed?

Reply:
Indeed not needed.